adcm_aio_client/core/objects/cm.py

```diff
@@ -110,3 +111,115 @@ def get_own_path(self: Self) -> Endpoint:
 
 class ComponentsNode(NonPaginatedChildAccessor[Service, Component, None]):
     class_type = Component
+
+
+class Prototype[ParentObject: InteractiveObject](InteractiveChildObject[ParentObject]):
+    @property
+    def id(self: Self) -> int:
+        return int(self._data["id"])
+
+    @property
+    def name(self: Self) -> str:
+        return str(self._data["name"])
+
+    @property
+    def display_name(self: Self) -> str:
+        return str(self._data["displayName"])
+
+    @property
+    def description(self: Self) -> str:
+        return str(self._data["description"])
+
+    @property
+    def type(self: Self) -> str:
```

Comment:
If there aren't requirements for it to be a string, probably better to introduce `Enum`